src/views/tutorial-view/utils/get-collection-context.ts

```diff
@@ -78,7 +78,7 @@ export function getCollectionContext(
 ): CollectionContext {
   const featuredIn = collectionCtx.featuredIn
     .filter(
-      (c) => c.id !== currentCollection.id && !c.slug.includes('onboarding') // filter out onboarding content for CS team
```

Comment:
Moving this filtering up to the frontend so that the featured collections can be consistent (including the current)